src/torrent.cpp

```diff
@@ -2720,6 +2729,15 @@ namespace {
 			valid_endpoints++;
 		});
 
+#if TORRENT_USE_RTC
+		std::string const protocol = url.substr(0, url.find(':'));
+		if (protocol == "ws" || protocol == "wss")
+		{
+			// WebSocket trackers will ignore the endpoint anyway
+			valid_endpoints = std::min(valid_endpoints, std::size_t(1));
```

Comment:
It's actually `min`, I put this here to clamp the number of endpoints for the websocket tracker to one, since you only need one request to it irrelevant of endpoints.

It's a bit clumsy, do you have another idea to achieve this?